release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,27 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-storage"]
+=== Storage
+
+[id="ocp-4-6-csi-driver-cso"]
+==== CSI drivers now managed by the Cluster Storage Operator
+The Container Storage Interface (CSI) drivers for AWS Elastic Block Store (EBS), Red Hat Virtualization (oVirt), and OpenStack Manila shared file system service are now managed by the Cluster Storage Operator in the {product-title} back end.
+
+For AWS EBS and oVirt, this feature installs the CSI Driver Operator and driver in the `openshift-cluster-csi-drivers` namespace by default. For Manila, the CSI Operator and driver are installed in the `openshift-manila-csi-driver` namespace.
+
+[IMPORTANT]
+====
+If you installed the AWS EBS or oVirt CSI Operator and driver on an {product-title} 4.5 cluster, you must uninstall them before you update to a newer version of {product-title}.
+====
+
+[id="ocp-4-6-lso-automation"]
+==== Automatic device discovery and provisioning with the Local Storage Operator
+The Local Storage Operator now has the ability to:
+
+* Automatically discover a list of available disks in a cluster. You can select a list of nodes, or all nodes, for auto-discovery to continuously run on.
+* Automatically provision local persistent volumes from attached devices. Appropriate devices are filtered and local volumes are created based on the filtered devices.
+
```

Comment:
It is much more clear for adding "local volume discovery" in first paragraph and "local volume set" in second paragraph.

Reply:
Thanks, that's helpful, @chao007. I've switched the order. I also changed the wording for auto-discovery: 
"for auto-discovery to continuously run on" --> "for auto-discovery to be continuously applied to"

Does this look good to you?

Reply:
No need to switch the order.  Sorry for not express clear.

Reply:
In this sentence "and local volumes are created based on the filtered devices."  " local volumes " is better to update to "PVs are provisioned"

Reply:
Thanks, @chao007 - does this look good to you now?